app/views/shared/_navigation.html.haml

```diff
@@ -0,0 +1,6 @@
+%nav
+  %ul.govuk-header__navigation
+    -if defined?(current_user) && current_user
```

Comment:
It's a pity that `current_user` doesn't exist everywhere so this could be `if current_user` instead, but this works 👍